components/html-block/README.md

```diff
@@ -68,14 +68,15 @@ To use `d2l-html-block` within another Lit component, use the [unsafeHTML](https
 
 ### Rendering MathML and LaTeX
 
-Examples are provided to display how user-authored math can be embedded within your webpage.
+Examples are provided to display how user-authored math can be embedded within your webpage. Note that rendering math requires the `mathjax` context to be set correctly. For testing and/or demo pages, you can import `@brightspace-ui/core/tools/mathjax-test-context.js` to set this context for you.
```

Comment:
I might be extra explicit and say "For testing and/or demo pages ONLY, ...".  I am worried that even with the console warning, people are going to import this in their non-demo code.

Reply:
Good point. I'll make that change.